### ISSUE TITLE
Add list of maintainers and approvers

### ISSUE THREAD

**Issue by arminru** [open-telemetry/opentelemetry-rust#844]
Hi!
I noticed that the maintainers and approvers of OTel Rust are neither listed in https://github.com/open-telemetry/community/blob/main/community-members.md nor here in the repository.

https://github.com/open-telemetry/opentelemetry-rust/blob/ae00d344a918d880c1ab28002a5c907f5d685e0e/CONTRIBUTING.md?plain=1#L124-L126

just links to

https://github.com/open-telemetry/opentelemetry-rust/blob/ae00d344a918d880c1ab28002a5c907f5d685e0e/CODEOWNERS#L1-L5

but users who are not part of the `open-telemetry` organization don't have access to view who's part of the rust-approvers (or rust-maintainers) team.

Please mention them in one of these places.
Thank you!

**Comment by arminru** [comment]
ping @open-telemetry/rust-maintainers @open-telemetry/rust-approvers

**Comment by hdost** [comment]
@TommyCpp @jtescher @djc @iredelmeier 
Thoughts ?

**Comment by TommyCpp** [comment]
I guess it's a reasonable ask. The question is shall we include company info? or just github handle + github display name. See Golang SIG's example [here](https://github.com/open-telemetry/opentelemetry-go/blob/main/CONTRIBUTING.md#approvers-and-maintainers)

**Comment by djc** [comment]
Just GitHub handle + plus some form of human name would be good enough IMO.

**Comment by hdost** [comment]
Something like

```
### Maintainers

* @djc - Dirkjan Ochtman
* @iredelmeier - Isobel Redelmeier
* @jtescher - Julian Tescher
* @TommyCpp - Zhongyang Wu

### Approvers

* @hdost - Harold Dost
* @frigus02 - Jan Kühle
* @awiede - Andreas Wiede
* @MikeGoldsmith - Mike Goldsmith
```
Sound good?

**Comment by djc** [comment]
I'd be curious what the most recent activity from each of those people has been? I'm clearly not the most active maintainer/approver but for some these names I don't recall seeing any activity. Maybe prune the list before publishing it anywhere?

**Comment by hdost** [comment]
Pruned version. My heuristic was more than 100 lines and more than 1 commit 😓 for the Emeritus list, but with no active in at least a year.

```
### Maintainers

* @djc - Dirkjan Ochtman
* @jtescher - Julian Tescher
* @TommyCpp - Zhongyang Wu

### Approvers
* @hdost - Harold Dost

### Emeritus
* @frigus02 - Jan Kühle
* @iredelmeier - Isobel Redelmeier
```

**Comment by frigus02** [comment]
I haven't had the time and energy to contribute to OpenTelemetry for a while now. Might even make sense to remove myself from rust approvers/maintainers.

**Comment by TommyCpp** [comment]
LGTM. Thanks for putting it together.

**Comment by hdost** [comment]
> I'd be curious what the most recent activity from each of those people has been? I'm clearly not the most active maintainer/approver but for some these names I don't recall seeing any activity. Maybe prune the list before publishing it anywhere?

Would it make sense to prune the existing members of the two @open-telemetry/rust-approvers  and @open-telemetry/rust-maintainers lists then too.

**Comment by cijothomas** [comment]
I think this can be closed now as #973 fixed this.
